docs/spark-standalone.md

```diff
@@ -374,6 +374,24 @@ To run an interactive Spark shell against the cluster, run the following command
 
 You can also pass an option `--total-executor-cores <numCores>` to control the number of cores that spark-shell uses on the cluster.
 
+#Spark Properties
+
+Spark applications supports the following configuration properties specific to standalone Mode: 
+<table class="table">
+  <tr><th style="width:21%">Property Name</th><th>Default Value</th><th>Meaning</th><th>Since Version</th></tr>
+  <tr>
+  <td><code>spark.standalone.submit.waitAppCompletion</code></td>
+  <td><code>false</code></td>
+  <td>
+  In Standalone cluster mode, controls whether the client waits to exit until the application completes.
```

Comment:
Standalone -> standalone
reporting -> polling

Reply:
Updated in the latest commit.